src/libs/localFileDownload/types.ts

```diff
@@ -0,0 +1,5 @@
+type LocalFileDownload = {
+    fileName: string;
+    textContent: string;
+};
```

Comment:
```suggestion
type LocalFileDownload = (fileName: string, textContent: string) => void;
```

It's better if we type the whole function signature and use it in the other files, for example:

```ts
// index.ts
import * as FileUtils from '../fileDownload/FileUtils';
import LocalFileDownload from './types';

const localFileDownload: LocalFileDownload = (fileName, textContent) => {
    const blob = new Blob([textContent], {type: 'text/plain'});
    const url = URL.createObjectURL(blob);
    const link = document.createElement('a');
    link.download = FileUtils.appendTimeToFileName(`${fileName}.txt`);
    link.href = url;
    link.click();
};

export default localFileDownload;
```